bin/cfg/TR1X_gameflow.json5

```diff
@@ -7,8 +7,8 @@
     "main_menu_picture": "data/titleh.png",
     "savegame_fmt_legacy": "saveati.%d",
     "savegame_fmt_bson": "save_tr1_%02d.dat",
-    "force_disable_game_modes": false,
-    "force_enable_save_crystals": false,
+    "force_game_modes": null,
+    "force_save_crystals": false,
```

Comment:
I think this should be `null` - with `false` there is no way to play with save crystals without manually changing this line (so the config option is ignored).

Reply:
That's correct, my mistake with copypasting.